epersons.md

```diff
@@ -163,8 +167,7 @@ NOTE: The new password is currently returned after an update but this could be r
 The currently authenticated user can modify their EPerson metadata. An administrator can modify any EPerson's metadata.  
 This includes, but is not limited to, last name, first name, phone, language
 
-## Create new EPerson
-
+## Create new EPerson as admin
```

Comment:
I originally read this as if you implied it was a way to create an Admin User account.  I think we should reword this to something like "Create new EPerson (requires admin permissions)"  or something similar.

Reply:
Adjusted the wording